wa/framework/configuration/plugin_cache.py

```diff
@@ -79,6 +79,11 @@ def add_configs(self, plugin_name, values, source):
             msg = "Source '{}' has not been added to the plugin cache."
             raise RuntimeError(msg.format(source))
 
+        if caseless_string(plugin_name) == 'global':
+            msg = '"global" section not allowed in configuration. Please move '\
+            'contents to the top level "config" entry.'
```

Comment:
This nneed to be rephrased. Something a long the lines of

> global/config entry specified inside config section; If this is  inside a config file, move the its contents into the top level.

Hm, dunno that's clear enough....

Reply:
Does this read any better?